core/src/main/scala/cats/instances/queue.scala

```diff
@@ -17,6 +19,17 @@ trait QueueInstances extends cats.kernel.instances.QueueInstances {
 
       def combineK[A](x: Queue[A], y: Queue[A]): Queue[A] = x ++ y
 
+      override def combineAllOptionK[A](as: IterableOnce[Queue[A]]): Option[Queue[A]] = {
+        val iter = as.iterator
+        if (iter.isEmpty) None else Some(appendAll(iter, Queue.newBuilder[A]).result())
+      }
+
+      override def fromIterableOnce[A](as: IterableOnce[A]): Queue[A] = {
+        val builder = Queue.newBuilder[A]
+        builder ++= as.iterator
```

Comment:
do we need `.iterator`? won't `builder ++= as` do?

Reply:
Thanks! Think I got lost in a see of different versions with different APIs. I've fixed this